dashboard/app/views/layouts/application.html.haml

```diff
@@ -21,6 +21,8 @@
     = stylesheet_link_tag    'css/code-studio', media: 'all'
     = stylesheet_link_tag    'application', media: 'all'
     = render partial: 'layouts/analytics'
+    // data needed for dcdo.js
+    %script{data: {'dcdo': DCDO.frontend_config.to_json}}
```

Comment:
oops, also, this doesn't look like the right syntax for a haml comment. please use `-#`

Reply:
sorry to ramble on here, but I see now that you've copied existing comment syntax in this file. the existing comments appear in the page we send down to the end user, which we probably don't want. this is what I see in the chrome debugger on studio.code.org/home today:
<img width="618" alt="Screen Shot 2021-12-09 at 11 29 28 AM" src="https://user-images.githubusercontent.com/8001765/145463176-d24da872-1d8b-45d6-8b73-4c97451d1c3f.png">

my suggestion would still be to use the `-#` haml comment syntax here, and it can be up to you whether you want to update other comment syntax in this file to match. none of those comments really need to get sent to the browser, and since they are included on every page load, it seems worth leaving unnecessary bytes out of this file.

Reply:
Nice catch :)